modules/data.atmosphere/README.md

```diff
@@ -26,13 +26,15 @@ The PEcAn.data.atmosphere package is 'standalone'.
 
 ## Documentation
 
-* [Package Documentation](https://pecanproject.github.io/pecan//modules/data.atmosphere/inst/web/index.html)
+## * [Package Documentation](https://pecanproject.github.io/pecan//modules/data.atmosphere/inst/web/index.html)
```

Comment:
Let's use this link -- it's currently 404 because of a deployment hiccup, but should be the right one to use here when it comes online: 
```suggestion
* [Package Documentation](https://pecanproject.github.io/package-documentation/develop/data.atmosphere/index.html)
```